plutus-tx-plugin/test/Plugin/Data/recursive/interListConstruct.tplc-read.golden

```diff
@@ -0,0 +1,265 @@
+(program 1.1.0 ((/\(InterList_i0 :: * -> * -> *) ->
+                   \(InterCons_i0
+                       : all a_i0
+                             b_i0.
+                           a_i2 -> b_i1 -> InterList_i4
```

Comment:
Should we view a repeated `->` as an n-ary construct and break it consistently? Otherwise we get this kind of  thing which is pretty ugly.

Reply:
Yes, we should.